applications/StructuralMechanicsApplication/python_scripts/assign_nodal_elements_to_nodes_process.py

```diff
@@ -0,0 +1,135 @@
+        if settings.Has("rayleigh_damping"):
+            settings.SetValue("rayleigh_damping", to_validate_parameters["rayleigh_damping"])
+        else:
+            settings.AddValue("rayleigh_damping", to_validate_parameters["rayleigh_damping"])
+        if settings.Has("interval"):
+            settings.SetValue("interval", to_validate_parameters["interval"])
+        else:
+            settings.AddValue("interval", to_validate_parameters["interval"])
+
+        # List of auxiliar parameters to assign in case not defined
+        auxiliar_parameters = KratosMultiphysics.Parameters("""
+        {
+            "nodal_mass"                     : null,
+            "nodal_inertia"                  : [null, null, null],
+            "nodal_stiffness"                : [null, null, null],
+            "nodal_rotational_stiffness"     : [null, null, null],
+            "nodal_damping_ratio"            : [null, null, null],
+            "nodal_rotational_damping_ratio" : [null, null, null]
+        }
+        """
+        )
+
+        if not settings.Has("nodal_mass"):
+            settings.AddValue("nodal_mass", auxiliar_parameters["nodal_mass"])
+        if not settings.Has("nodal_inertia"):
+            settings.AddValue("nodal_inertia", auxiliar_parameters["nodal_inertia"])
+        if not settings.Has("nodal_stiffness"):
+            settings.AddValue("nodal_stiffness", auxiliar_parameters["nodal_stiffness"])
+        if not settings.Has("nodal_rotational_stiffness"):
+            settings.AddValue("nodal_rotational_stiffness", auxiliar_parameters["nodal_rotational_stiffness"])
+        if not settings.Has("nodal_damping_ratio"):
+            settings.AddValue("nodal_damping_ratio", auxiliar_parameters["nodal_damping_ratio"])
+        if not settings.Has("nodal_rotational_damping_ratio"):
+            settings.AddValue("nodal_rotational_damping_ratio", auxiliar_parameters["nodal_rotational_damping_ratio"])
+
+        # The main model part
+        self.model = Model
+        self.main_model_part = self.model[settings["model_part_name"].GetString()]
+
+        # The creation of the process
+        process_parameters = KratosMultiphysics.Parameters("""{}""")
+        process_parameters.AddValue("model_part_name", settings["sub_model_part_name"])
+        process_parameters.AddValue("rayleigh_damping", settings["rayleigh_damping"])
+        process_parameters.AddValue("nodal_mass", settings["nodal_mass"])
+        process_parameters.AddValue("nodal_inertia", settings["nodal_inertia"])
+        process_parameters.AddValue("nodal_stiffness", settings["nodal_stiffness"])
+        process_parameters.AddValue("nodal_rotational_stiffness", settings["nodal_rotational_stiffness"])
+        process_parameters.AddValue("nodal_damping_ratio", settings["nodal_damping_ratio"])
+        process_parameters.AddValue("nodal_rotational_damping_ratio", settings["nodal_rotational_damping_ratio"])
+        process_parameters.AddValue("interval", settings["interval"])
```

Comment:
how can this have an interval? You either create the new elements or you dont

Reply:
I guess I should remove the elements in that case

Reply:
I just checked the C++ code and the elementes are deactivated when not in the interval